sklearn/model_selection/_split.py

```diff
@@ -913,6 +915,177 @@ def get_n_splits(self, X, y, groups):
         return int(comb(len(np.unique(groups)), self.n_groups, exact=True))
 
 
+class BaseRepeatedCrossValidator(with_metaclass(ABCMeta)):
+    """Base class for repeated cross validators
+
+    Implementations must define `_get_cv`.
```

Comment:
This seems inelegant to me. A mixin is an option, perhaps. Otherwise, see my main comment.